docker-compose.yml

```diff
@@ -25,7 +25,7 @@ services:
     network_mode: host
     working_dir: /mini-lab
     dns:
-      - 192.168.121.1
+      - 172.17.0.1
```

Comment:
Could be already taken by a user who already started something else through docker-compose.